lib/serre.gi

```diff
@@ -394,7 +399,11 @@ InstallMethod( REPN_ComputeUsingSerre, "for linear reps", [ IsGroupHomomorphism
     end;
 
     parallel := ValueOption("parallel");
-
+    if IsBoundGlobal("ParListByFork") then
+        ParListByFork := ValueGlobal("ParListByFork");
+    elif parallel <> fail then
+        Error("I need the package IO >= 4.7.0 to use the parallel option!");
```

Comment:
This check does not actually check for IO 4.7.0 anymore. In fact I am not sure why the code below does that, as `ParListByFork` predates 4.7.0 by a lot?!?

So how about this:
```suggestion
        Error("The GAP package IO must be loaded to use the parallel option!");
```